CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+## Enhancements
```

Comment:
Let me know if this should be Breaking Changes instead, I wasn't sure if we flag changes like this as breaking for features that aren't GA yet.

Reply:
Oh, looking at the last release's notes, this should be a breaking change, will update this.